src/pages/ArticleDetailPage.tsx

```
@@ -0,0 +1,157 @@
+              <span className="flex items-center gap-1">
+                <Clock className="h-4 w-4" />
+                {article.readingTime} min lees
+              </span>
+            )}
+          </div>
+        </header>
+
+        {/* Content */}
+        <div
+          className="prose prose-lg max-w-none mb-12"
+          dangerouslySetInnerHTML={{ __html: article.content }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)